### ISSUE TITLE
Ctrl-r binding issue: command not found

### ISSUE THREAD

**Issue by elaye** [junegunn/fzf#1367]
- Category
    - [ ] fzf binary
    - [ ] fzf-tmux script
    - [x] Key bindings
    - [ ] Completion
    - [ ] Vim
    - [ ] Neovim
    - [ ] Etc.
- OS
    - [ ] Linux
    - [x] Mac OS X
    - [ ] Windows
    - [ ] Windows Subsystem for Linux
    - [ ] Etc.
- Shell
    - [x] bash
    - [ ] zsh
    - [ ] fish

I just installed `fzf 0.17.4` and I use it with `bash` in `iTerm2 3.2.0`. I have an issue with the `Ctrl-r` key binding. When I do it, I have the following error message:

```
`__ffhistory__`-bash: __ffhistory__: command not found
```
And the current prompt line has `_z` printed on it. Here's a screenshot:

<img width="395" alt="screen shot 2018-08-31 at 09 38 30" src="https://user-images.githubusercontent.com/2633171/44899345-c2d34b80-ad01-11e8-9a54-c8c32708fb94.png">

**Comment by elaye** [comment]
Yes, the `__ffhistory__` function doesn't exist. However the `__fzf_history__` function does exist (https://github.com/junegunn/fzf/blob/8c6fcee3caef11eff4875e98d98debca7cef8df2/shell/key-bindings.bash#L54). The fact that the two missing characters from the function name are printed on the next prompt line when I hit `Ctrl-r` seems really strange.

**Comment by doneel** [comment]
@elaye Are you using vi mode in bash? If you have something like 'ff' or 'fd' as your movement mode trigger, bash going to interpret the first `f` in ` __fzf_history__() ( ` as a potential `vi-movement-mode` command.

You can see we delete the line, enter insert mode, and then input `__f` while in insert mode, so bash 
https://github.com/junegunn/fzf/blob/8540902a35361dba217cc3ad86ff2215ae51918e/shell/key-bindings.bash#L106

This normally cause a noticeable delay before bash prints the next character, but I don't have a good explanation for how it causes the missing two characters to appear on the next line and I don't have a good explanation why this doesn't affect everyone who has a `vi-movement-mode` binding starting with any of the characters in the command, which seems like it'd be a lot of people.

In the meantime I worked around this find/replacing the function names in `key-bindings.bash` which is unsatisfying but works.

**Comment by elaye** [comment]
Ah yes good call ! I have `bind '"fd":vi-movement-mode'` in my `.bash_profile` and when I comment it it works normally. Thanks @doneel !
